.gitignore

```diff
@@ -10,3 +10,4 @@ RUNNING_PID
 .secrets
 node_modules
 bundle.js
```

Comment:
thought (non-blocking): We should be able to remove `bundle.js` from here now. But we'd probably want to make sure everyone deletes any `bundle.js` they still have, so as not to accidentally check it in. We can remove it in a follow-up PR after this one is merged to allow a small transition period.

Reply:
Sounds good!